hasher-matcher-actioner/hmalib/indexers/s3_indexers.py

```diff
@@ -35,6 +36,8 @@ class S3BackedInstrumentedIndexMixin:
     Also overrides methods to wrap instrumentation with hmalib.metrics library.
     """
 
+    INDEXES_PREFIX = "index/"
```

Comment:
(nonblocking) We haven't been consistent about this being a variable vs just locally defining it. It used to be something defined in terraform...

Reply:
nevermind, looks like this is a step in the right direction regardless